nmoscommon/webapi.py

```diff
@@ -801,7 +801,7 @@ def error(self, e):
             if isinstance(e, AuthlibBaseError):
                 response = {
                     'code': 400,
-                    'error': e.description,
+                    'error': e.message if e.description else e.error,
```

Comment:
Any reason why e.description is checked here but then e.message used?

Reply:
`e.message` uses a combination of both the error and description.. it basically avoids having colons at the end of error messages that dont have descriptions